optimal_transport_morphometry/core/rest/image.py

```diff
@@ -2,7 +2,7 @@
 from django.http import HttpResponseRedirect
 from django.shortcuts import get_object_or_404
 from django_filters import rest_framework as filters
-from drf_yasg2.utils import swagger_auto_schema
+from drf_yasg.utils import swagger_auto_schema
```

Comment:
@brianhelba I haven't kept up here, what's the story on drf_yasg?

Reply:
@zachmullen It was unmaintained, then forked as `drf_yasg2`. About a week after the fork, the original maintainer showed up again, and it was agreed by all parties to unfork.

Just use `drf_yasg` in the future. They are accepting PRs if we find issues.